examples/chat_stream/README.md

```
@@ -0,0 +1,35 @@
+
+The example supports configuration via environment variables:
+
+```bash
+# Set Meilisearch server URL (optional, defaults to http://localhost:7700)
+export MEILI_HOST="http://localhost:7700"
+
+# Set API key (optional, but recommended for production)
+export MEILI_API_KEY="your-api-key"
+```
+
+## Configuration
```

markdownlint-cli2 (0.17.2)

Multiple headings with the same content

(MD024, no-duplicate-heading)